src/zoo/owl_zoo_log.ml

```diff
@@ -0,0 +1,85 @@
+(*
+ * OWL - an OCaml numerical library for scientific computing
+ * Copyright (c) 2016-2017 Liang Wang <liang.wang@cl.cam.ac.uk>
+ *)
+
+let dir = Sys.getenv "HOME" ^ "/.owl/zoo/"
+let log = dir ^ "log.htb"
+
+let _syscall cmd =
+  let ic, oc = Unix.open_process cmd in
+  let buf = Buffer.create 50 in
+  (try
+     while true do
+       Buffer.add_channel buf ic 1
+     done
+   with End_of_file -> ());
+  Unix.close_process (ic, oc);
+  Buffer.contents buf
+
+let _create_log () =
```

Comment:
log often means logging everything, but i guess you are mostly dealing with versions here ... I am not sure log is a good name here, but i also ack i don't have a good suggestion yet, can think further.

Reply:
How about `zoo_version_db`? Or since it's already in the `~/.owl/zoo` directory, so maybe just `version_db`?

Reply:
up to you, but i do find log is not very obvious for its purpose, can be short as `zoo_ver` or surreal as `zoo_keeper` :)